pyvista/core/filters/data_set.py

```diff
@@ -2511,6 +2511,14 @@ def glyph(
             ``indices``. The values of the range (see ``rng``) affect lookup
             in the table.
 
+            .. note::
+
+                The provided geometry should initially be "pointing" in the
```

Comment:
Should we say that the `The reference direction is relative to (1, 0, 0). It is recommended that the provided geometry should be "pointing"...` as the first sentence?  It can be valid for the provided geometry to not be "pointing" in the positive "+x" direction if the vectors are also transformed in the same way.  This is complicated, but it can be done.

Reply:
Another thought: I'm not sure if there is a better way to explain that there is not a true "pointing" towards `(1, 0, 0)` for a mesh unless it is 1D (even then it is pointing in two directions).  It is all about the reference direction you want to transform via the vector.  Suggestion for wording could be `The reference direction is relative to (1, 0, 0) on the provided geometry. That is, the provided geometry will be rotated from (1, 0, 0) to the direction of the orient vector at each point.`